src/layouts/components/Header.tsx

```diff
@@ -191,25 +191,7 @@ function HeaderLink({
       >
         <span
           data-label={label}
-          className="
-            text-transparent
-            font-medium
-            before:bg-black-900
-            relative
-            before:gradient-text
-            before:font-medium
-            before:opacity-100
-            group-hover:before:opacity-0
-            before:leading-[20.8px]
-            after:leading-[20.8px]
-            after:gradient-text
-            after:font-medium
-            after:brand-gradient-1
-            after:opacity-0
-            group-hover:after:opacity-100
-            before:content-[attr(data-label)]
-            after:content-[attr(data-label)]
-          "
+          className="font-medium brand-gradient-1 group-hover:text-transparent bg-clip-text transition-all ease-in duration-300"
```

Comment:
currently it does have transition inside `gradient-text` that I removed. I just updated css to simplify and still achieve the same result 🤔 i was hoping to avoid the use of before/after selectors as much as possible, and reserve them for future use (like adding icons).

But ya not related to button at all, I can totally leave this untouched. Let me know if I should revert :)

Reply:
but i noticed a diff in on hover behaviour after this change, can you check again?

Reply:
Ohh, right it is slower in this PR. Pushing an update